tests/e2e/project/commit.spec.ts

```diff
@@ -61,7 +63,7 @@ test("created file", async ({ page }) => {
     `${projectFixtureUrl}/remotes/${bobRemote}/commits/d6318f7f3d9c15b8ac6dd52267c53220d00f0982`,
   );
   await expect(
-    page.locator("text=1 file(s) created with 9 addition(s) and 0 deletion(s)"),
+    page.locator("text=1 file(s) added with 9 insertion(s) and 0 deletion(s)"),
```

Comment:
why not keep it as "created"? you've kept the one in [`FileDiff.svelte`](https://github.com/radicle-dev/radicle-interface/pull/565/files#diff-86922c6cf991e9927a1a37a653efc6e9c1db5dfacb4a99a772798d30d4b0f3dbR122) as created.

This should make `visual` pass as well?

Reply:
Ah, true. Will do.

Reply:
Trying to match the changes in #515.

Reply:
I see `visual` still failing, why is that?

Reply:
Because the wording changed:

> The visual diff failure is expected, renamed created to added.

Reply:
I'm aware of that. I thought you reverted it back in this [diff](https://github.com/radicle-dev/radicle-interface/compare/208c156f28fe482b197782d1b1c742a36e53f985..12c10e26dd47dc86d87564956e3d286b4688f02d)?

Reply:
I changed my mind : ) and reverted it back. I had another look at #515 and wanted to match what we're doing there, so we have these changes in `master`.

Reply:
Once it's merged, the snapshots will update and that specific test won't fail in #515 anymore.